app/views/admin/applicants/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Applicant Name: <%= @applicant.name %></h2>
+<p>Address:<br/>
+<%= @applicant.street_address %><br/>
+<%= @applicant.city %>, <%= @applicant.state %> <%= @applicant.zip_code %></p>
+<p>Why I would make a good home:<br/>
+<%= @applicant.description %></p>
+<p>Application Status:<br/>
+<%= @applicant.application_status %></p>
+
+<h3>Pets being applied for:</h3>
+<ul>
+<% @applicant_pets.each do |pet| %>
```

Comment:
here is where you can remove the `@applicant_pets` instance variable from the controller and just call `@applicant.pets`.